gamedata/alldefs_post.lua

```diff
@@ -169,6 +172,29 @@ function UnitDef_Post(name, uDef)
 			end
 		end
 
+		if modOptions.unit_restrictions_notech15 then
+			-- Tech 1.5 is a semi offical thing, modoption ported from teiserver meme commands
+			local tech15 = {
+				corhp		= true,
```

Comment:
these units could get some sort of `tech_1.5` customparam

Reply:
also the table should be outside the function since now you're making a new instance of the table and filling it over each time

Reply:
perhaps things could get a numerical tech level customparam in general. could mostly be autocalculated, but a few exceptions like these 1.5 would be set manually

Reply:
🌏👨‍🚀🔫👨‍🚀
https://github.com/search?q=repo%3Abeyond-all-reason%2FBeyond-All-Reason%20techlevel&type=code

Reply:
> these units could get some sort of tech_1.5 customparam

I haven't been able to comprehend the total covrage of techlevel custom param to see if it would be safe to store it as anything than a full intiger.

> table should be outside the function

It's done like that throughout the file several times for other modoptions, if such is bad it probably best be adjusted for all of them.